lib/ansible/module_utils/tacp.py

```diff
@@ -347,3 +347,52 @@ def get_resource_by_uuid(uuid, component, api_client):
     #                 uuid=datacenter.uuid, uuid)
     #         except ApiException as e:
     #             return "Exception when calling get_firewall_profiles_using_get"
+
+
+class ApplicationGroupResource(object):
+    def __init__(self, api_client, name, datacenter_uuid=None):
+        self.api = tacp.ApplicationGroupsApi(api_client)
+        self.name = name
+        self.datacenter_uuid = datacenter_uuid
+
+    def get(self):
+        """ Returns the application group as a dict or None if nothing found.
+
+        eg: {'datacenter_uuid': '5b665e17-071d-4159-a0e0-58f4081ccd73',
+             'name': 'gulf',
+             'uuid': 'cef6c09c-d371-408e-8fd6-292befcc42d6'}
+
+        """
+        filters = f'name=={self.name}'
```

Comment:
We need to support python 2 in this code, so we cannot use f-strings

Reply:
Modified